lib/cocoapods-downloader/remote_file.rb

```diff
@@ -0,0 +1,179 @@
+require 'zlib'
+require 'fileutils'
+require 'uri'
+
+module Pod
+  module Downloader
+    class RemoteFile < Base
```

Comment:
this probably needs tests

Reply:
moved some tests from http_specs.rb to remote_file_spec.rb.